pybmix/core/pybmixcpp/python_embedding/includes.h

```diff
@@ -0,0 +1,6 @@
+#ifndef BAYESMIX_INCLUDES_2_H_
+#define BAYESMIX_INCLUDES_2_H_
+
+#include "load_py_hierarchies.h"
+
+#endif  // BAYESMIX_INCLUDES_2_H_
```

Comment:
We didn't notice, would ```PYTHON_EMBEDDING_INCLUDES_H_``` or ```PY_LOADERS_INCLUDES_H_``` be OK? Also notice that at the moment it only includes ```load_py_hierarchies.h``` but it will also include ```load_py_mixings.h``` in the next PR for the mixing

Reply:
Sure. In general, it is sufficient that the name is specific to the file without the chance of duplicate names. That's why we usually stick to `PATH_TO_FILE_H_`. Both of those are fine, but I like the first one better